### PR TITLE
Change from drawer layout to navbar layout

### PR DESCRIPTION
**Pull request by tilokowalski**

### CODE REVIEW

**Review by thisami — CHANGES_REQUESTED** [review]
Nö, approve ich nicht

File: src/main/java/com/fes/hazelnut/schoolnut/view/MainView.java [hunk 1]
```diff
@@ -1,245 +1,240 @@
 package com.fes.hazelnut.schoolnut.view;
 
-import com.fes.hazelnut.schoolnut.data.entity.user.UserEntity;
+import java.util.concurrent.CompletableFuture;
+
+import com.fes.hazelnut.schoolnut.helper.ServiceHelper;
```
Comment (tilokowalski) [inline src/main/java/com/fes/hazelnut/schoolnut/view/MainView.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.fes.hazelnut.schoolnut.helper.ServiceHelper'

Comment (tilokowalski) [inline src/main/java/com/fes/hazelnut/schoolnut/view/MainView.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.fes.hazelnut.schoolnut.helper.ServiceHelper' import. Should be before 'java.util.concurrent.CompletableFuture'.

File: src/main/java/com/fes/hazelnut/schoolnut/view/MainView.java [hunk 2]
```diff
@@ -1,245 +1,240 @@
...
+import com.fes.hazelnut.schoolnut.helper.session.NavigationHelper;
```
Comment (tilokowalski) [inline src/main/java/com/fes/hazelnut/schoolnut/view/MainView.java:6]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.fes.hazelnut.schoolnut.helper.session.NavigationHelper' import. Should be before 'java.util.concurrent.CompletableFuture'.

**Review by thisami — APPROVED** [review]

**Review by lenav99 — APPROVED** [review]

**Review by thisami — APPROVED** [review]

**Pull request merged**